modules/ROOT/pages/troubleshoot-visualizer.adoc

```diff
@@ -31,6 +31,10 @@ If you encounter an error or issue while using Visualizer, you can try these sol
 |For CloudHub applications, you must use an `-AM` Mule to support Anypoint Monitoring, or upgrade to a Mule 4 version available after March 22, 2019, or Mule 3.9 version available after April 5, 2019, and then enable the Monitoring agent. View a list of xref:setup.adoc[supported Mule versions].
 |===
 
+== Why can't I see metrics for my connections?
+
+Metrics at the connection level are only available for certain types of connection. For more information on edge metrics compatibility, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].
```

Comment:
NOTE "at the connection level". I think we want to change all instances of "on the connection level" to "at". Please search for and evaluate.